--- notify-4.0.15/Cargo.toml	2020-01-07T14:33:26+00:00
+++ notify-4.0.15/Cargo.toml	2020-01-25T00:51:39.365338+00:00
@@ -40,7 +40,7 @@
 manual_tests = []
 timing_tests = []
 [target."cfg(target_os=\"linux\")".dependencies.inotify]
-version = "^0.7"
+version = "^0.8"
 default-features = false
 
 [target."cfg(target_os=\"linux\")".dependencies.mio]
@@ -48,14 +48,6 @@
 
 [target."cfg(target_os=\"linux\")".dependencies.mio-extras]
 version = "^2.0.5"
-[target."cfg(target_os=\"macos\")".dependencies.fsevent]
-version = "0.4"
-
-[target."cfg(target_os=\"macos\")".dependencies.fsevent-sys]
-version = "2"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.8"
-features = ["fileapi", "handleapi", "ioapiset", "minwinbase", "synchapi", "winbase", "winnt"]
 [badges.travis-ci]
 branch = "main"
 repository = "notify-rs/notify"
